numba/lowering.py

```diff
@@ -188,7 +188,19 @@ def lower(self):
 
         if config.DUMP_LLVM:
             print(("LLVM DUMP %s" % self.fndesc).center(80, '-'))
-            print(self.module)
+            if config.HIGHLIGHT_DUMPS:
```

Comment:
document this?

Reply:
Yes, a short description for this env-var would do.

Reply:
done